src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -30,4 +30,11 @@ class HttpExecutionServiceDAO( executionServiceURL: String )( implicit system: A
     val pipeline = addHeader(Cookie(authCookie)) ~> sendReceive ~> unmarshal[ExecutionServiceStatus]
     Await.result(pipeline(Get(url)),Duration.Inf)
   }
+
+  override def outputs(id: String, authCookie: HttpCookie): ExecutionServiceOutputs = {
+    val url = executionServiceURL + s"/workflow/${id}/outputs"
+    import system.dispatcher
```

Comment:
If you're pulling in implicits (which obviously akka uses extensively) it's a way to make sure that the right stuff is being used in the right places. Otherwise I agree that imports should generally be at the top